kafka-ui-react-app/src/components/Connect/Details/Actions/__tests__/Actions.spec.tsx

```diff
@@ -35,8 +35,12 @@ const expectActionButtonsExists = () => {
 };
 const afterClickDropDownButton = async () => {
   const dropDownButton = screen.getAllByRole('button');
-  expect(dropDownButton.length).toEqual(1);
-  await userEvent.click(dropDownButton[0]);
+  expect(dropDownButton.length).toEqual(2);
```

Comment:
i don't think the click dropdown button should check how many dropdowns we have, now it is just clicking the first or the second button.

and if this click is for the delete , we should fetch it with its text Delete , as it is done in the Restart. and we can rename the function `afterClickDeleteButton`